### PR TITLE
valueref

### PR DESCRIPTION
**Pull request by mykytam**

### CODE REVIEW

**Comment by MentTea** [comment]
NiceWork

**Comment by MentTea** [comment]
NiceWork

**Comment by AlexanderGratch** [comment]
Лииишние файлы

File: module1/src/test/java/net/broscorp/valueref/FooTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class FooTest {
+
+  // Java is always pass-by-value
+
+  Foo object = new Foo();
+  int number = 1;
+  List<String> listOfStrings = new ArrayList<>();
```
Comment (AlexanderGratch) [inline module1/src/test/java/net/broscorp/valueref/FooTest.java:16]: Just a hint - better to init something in tests in a method annotated with @Before annotation. It's a more clear way

**Review by AlexanderGratch — APPROVED** [review]
good

**Pull request closed without merging**